scipy/stats/tests/test_entropy.py

```diff
@@ -0,0 +1,275 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.testing import assert_equal, assert_allclose
+# avoid new uses of the following; prefer assert/np.testing.assert_allclose
```

Comment:
Others have argued that `assert` is preferred over `assert_`. (see [here](https://github.com/scipy/scipy/issues/8583)). I think the issue was closed because we do not need to _change_ old uses of `assert_`, but since it _might_ have a slight edge and looks better, might as well use `assert` in new code.

NumPy recommends `assert_allcose` over `assert_almost_equal` and `assert_array_almost_equal`. (see [here](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html).)

Reply:
OK thanks for the references.